tinyagentos/tuiui_conduit.py

```
@@ -0,0 +1,402 @@
+
+def default_socket_path() -> str:
+    """Return the default apphost socket path.
+
+    Matches ``$XDG_RUNTIME_DIR/tuiui-$USER/apphost.sock`` per the spike
+    (per-user, mode 0600 socket, 0700 directory). Falls back to
+    ``/tmp/tuiui-$USER/apphost.sock`` when ``XDG_RUNTIME_DIR`` is unset.
+    """
+    user = os.environ.get("USER", "unknown")
+    if xdg := os.environ.get("XDG_RUNTIME_DIR"):
+        return os.path.join(xdg, f"tuiui-{user}", "apphost.sock")
+    return os.path.join("/tmp", f"tuiui-{user}", "apphost.sock")
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)